conversion/binary_to_octal.py

```diff
@@ -0,0 +1,29 @@
+def binary_to_octal(msg: str) -> str:
+    """
+    implementasi biner ke oktal
+    >>> binary_to_octal("1111")
+    '17'
+    """
+    if not all(char in "01" for char in msg):
+        raise ValueError("Nilai non-biner diteruskan ke fungsi")
+    if not msg:
+        raise ValueError("String kosong diteruskan ke fungsi")
+
+    oct_string = ""
+    while len(msg) % 3 != 0:
+        msg = "0" + msg
```

Comment:
```suggestion
        binary = "0" + binary
```
ubah variablenya ke ``binary``